librz/il/il_validate.c

```diff
@@ -583,7 +822,7 @@ static bool validate_pure(VALIDATOR_PURE_ARGS) {
  */
 RZ_API bool rz_il_validate_pure(RZ_NULLABLE RzILOpPure *op, RZ_NONNULL RzILValidateGlobalContext *ctx,
 	RZ_NULLABLE RZ_OUT RzILSortPure *sort_out, RZ_NULLABLE RZ_OUT RzILValidateReport *report_out) {
-	LocalContext local_ctx;
+	LocalContext local_ctx = { 0 };
```

Comment:
Don't add `= { 0 }` unless it is actually used that way.

Reply:
ok